.github/automation/.drone.yml

```diff
@@ -83,7 +83,7 @@ steps:
   - .github/automation/env/clang.sh
   - export CC=clang
   - export CXX=clang++
-  - .github/automation/build_acl.sh  --version 23.02.1 --arch armv8a --root-dir $(pwd)/ComputeLibrary
+  - .github/automation/build_acl.sh  --version 23.11 --arch armv8a --multi_isa --fixed_format_kernels --root-dir $(pwd)/ComputeLibrary
```

Comment:
We typically use `multi_isa` everywhere, so this is fine. As far as I know, this specific script isn't actually being run anywhere in CI because we don't have any AArch64 drone runners. It currently exists more as a reference, so this is just bitrot. We are working on getting some new public runners. Not drone ones though, so it doesn't matter too much either way.